app/models/project.rb

```diff
@@ -26,6 +32,7 @@ class Project < ApplicationRecord
   }
 
   belongs_to :language
+  belongs_to :product, optional: true
```

Comment:
why is this optional?

Reply:
This is because when we receive the webhooks from GH, if the project don't exist it is created. The payload does not have any information about the product and it fails to create it. Maybe can be change it to only find the `project` and the admin has to create the project manually before start reporting the events. WDYT?

Reply:
@enzofab91 for the future, I don't think it's fine to merge PRs with open discussions